about/release_policy.rst

```diff
@@ -90,15 +83,20 @@ on GitHub.
 +-------------+----------------------+--------------------------------------------------------------------------+
 | **Version** | **Release date**     | **Support level**                                                        |
 +-------------+----------------------+--------------------------------------------------------------------------+
-| Godot 4.0   | Q1 2023 (estimate)   | |unstable| *Beta.* Current focus of development (unstable).              |
+| Godot 4.1   | Q2-Q3 2023 (estimate)| |unstable| *Development.* Receives new features as well as bug fixes     |
+| (`master`)  |                      | while under development.                                                 |
++-------------+----------------------+--------------------------------------------------------------------------+
+| Godot 4.0   | March 2023           | |supported| Receives fixes for bugs, security and platform support       |
+|             |                      | issues, as well as backwards-compatible usability enhancements.          |
 +-------------+----------------------+--------------------------------------------------------------------------+
-| Godot 3.6   | Q1-Q2 2023 (estimate)| |supported| *Beta.* Receives new features as well as bug fixes while     |
-| (LTS)       |                      | under development. Will be released *after* 4.0.                         |
+| Godot 3.6   | Q2-Q3 2023 (estimate)| |supported| *Beta.* Receives new features as well as bug fixes while     |
+| (`3.x`, LTS)|                      | under development. Will be released *after* 4.0.                         |
 +-------------+----------------------+--------------------------------------------------------------------------+
 | Godot 3.5   | August 2022          | |supported| Receives fixes for bugs, security and platform support       |
 |             |                      | issues, as well as backwards-compatible usability enhancements.          |
 +-------------+----------------------+--------------------------------------------------------------------------+
-| Godot 3.4   | November 2021        | |partial| Receives fixes for security and platform support issues only.  |
+| Godot 3.4   | November 2021        | |eol| No longer supported as fully superseded by the compatible 3.5      |
```

Comment:
"No longer supported as fully superseded" hurts my brain a bit. But I guess we use it for other versions too.

Reply:
Could replace the "as" with a comma or just make it two sentences. IMO shouldn't be in scope for this PR though. :)